SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -86,6 +89,25 @@ static LogLevel PlatformLogLevel
             set => AppCenterLog.Level = value;
         }
 
+        /// <summary>
+        /// Is network requests allowed? True by the default.
```

Comment:
Let's avoid question marks in comments. I think something like "Allow or disallow network requests." (as you did in other parts) should work.